substrafl/remote/register/register.py

```diff
@@ -32,11 +32,10 @@
 
 # minimal and maximal values of Python 3 minor versions supported
 # we need to store this as integer, else "3.11" < "3.9" (string comparison)
-MINIMAL_PYTHON_VERSION = 8  # 3.8
+MINIMAL_PYTHON_VERSION = 9  # 3.9
```

Comment:
After thinking, should be done in a separate PR

Reply:
Agreed 👍